src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./utils/_reset";
+@import "./blocks/header.scss";
+@import "./blocks./card.scss";
+@import "./blocks./stars.scss";
+@import "./utils/_extends";
```

Comment:
Check the order:

1. Variables
2. Functions
3. Reset
4. Extends
5. ...
6. Blocks